t/app/controller/report_new.t

```diff
@@ -1494,10 +1495,17 @@ subtest "unresponsive body handling works" => sub {
             "submit good details"
         );
 
+        my $user = FixMyStreet::App->model('DB::User')->find( { email => $test_email } );
+        ok $user, "test user does exist";
+
         my $report = $user->problems->first;
         ok $report, "Found the report";
         is $report->bodies_str, undef, "Report not going anywhere";
 
+        my $email = $mech->get_email;
+        ok $email, "got an email";
+        like $email->body, qr/despite not being sent/i, "correct email sent";
```

Comment:
This should be covered, as the normal email test in the rest of this file is looking for a string that isn't in this email, iyswim.